apps/core/src/components/Inputs/SendTokenFormInput.tsx

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { ButtonPill, Input, InputType } from '@iota/apps-ui-kit';
+import { CoinStruct } from '@iota/iota-sdk/client';
+import { useGasBudgetEstimation } from '../../hooks';
+import { useEffect } from 'react';
+import { useField, useFormikContext } from 'formik';
+import { TokenForm } from '../../forms';
+
+export interface SendTokenInputProps {
+    coins: CoinStruct[];
+    symbol: string;
+    coinDecimals: number;
+    activeAddress: string;
+    to: string;
+    onActionClick: () => Promise<void>;
+    isMaxActionDisabled?: boolean;
+    name: string;
+}
+
+export function SendTokenFormInput({
+    coins,
+    to,
+    symbol,
+    coinDecimals,
+    activeAddress,
+    onActionClick,
+    isMaxActionDisabled,
+    name,
+}: SendTokenInputProps) {
+    const { values, setFieldValue, isSubmitting, validateField } = useFormikContext<TokenForm>();
+    const gasBudgetEstimation = useGasBudgetEstimation({
+        coinDecimals,
+        coins: coins ?? [],
+        activeAddress,
+        to: to,
+        amount: values.amount,
+        isPayAllIota: values.isPayAllIota,
+    });
+
+    const [field, meta, helpers] = useField<string>(name);
+    const errorMessage = meta.error;
+    const isActionButtonDisabled = isSubmitting || !!errorMessage || isMaxActionDisabled;
```

Comment:
```suggestion
            const isActionButtonDisabled = isSubmitting || isMaxActionDisabled;
```
we should remove the ` !!errorMessage` because max button is disabled if there is an error message
![image](https://github.com/user-attachments/assets/b6559e4a-1ea8-4d5c-9786-6419cff800ee)